.scalafmt.conf

```diff
@@ -4,3 +4,5 @@ align.preset = none # never align to make the fmt more diff friendly
 maxColumn = 100
 runner.fatalWarnings = true
 trailingCommas = multiple
+
+version = 2.7.5
```

Comment:
bikeshedding but maybe put it at the top given that it determines which of the following options are valid?

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8447/commits/bb0753c3289d3490969dc83304af1faf72ea5963